### PR TITLE
new features as a total, display songs 

### PR DESCRIPTION
**Pull request by davigrca**

### CODE REVIEW

File: how-to-use.html [hunk 1]
```diff
@@ -101,10 +101,10 @@
       <h2>TIPS FOR MAKING PLAYLISTS</h2>
       <ul>
         <li><span style="color:#c63d3d">CrossFaderYT does not work in YouTube's native Fullscreen mode.</span> When viewing your created playlist, use your browser's fullscreen mode (F11 on Windows, View -> Fullscreen on Mac OSX)</li>
-        <li>YouTube song IDs come from the URL of your video. <span style="color:#c63d3d">Use the Sample Playlist button to see how this works</span>.</li>
+        <li>YouTube song IDs come from the end of URL of your video. <span style="color:#c63d3d">Use the Sample Playlist button to see how this works</span>. You can click on ID to open the video.</li>
         <li>Start time and end times are in <span style="color:#c63d3d">seconds</span>. Leave this blank to use the entire video.</li>
-        <li>Start time and end times are in <span style="color:#c63d3d">seconds</span>. Leave this blank to use the entire video.</li>
-        <li>Contribute new features on GitHub :) See a link to the project in the footer</li>
+
+        <br>Contribute new features on GitHub :) See a link to the project in the footer</ul></br>
```
Comment (jcneally) [inline how-to-use.html:107]: There is a `</ul>` tag here and below as well, which is one too many

File: index.html [hunk 2]
```diff
@@ -433,8 +505,19 @@
       var songList = [];
       var selectedSongIndex;
 
-      //creating a function songDetail to check if it exists youtubeLink variable inside of div songDetail, if it does, so it returns id youtubeLink variable, it it doesnt, it reurns string empty "
-      var songDetail = (youtubeLink) => (`<div class="song-detail">${youtubeLink ? youtubeLink.id : ""} </div>`);
+      //creating a function songDetail to check if it exists youtubeLink variable inside of div songDetail, if it does, so it returns id youtubeLink variable, it it doesnt, it returns empty string"
+      /*var songDetail = function (youtubeLink) {
```
Comment (jcneally) [inline index.html:509]: Let's remove this unused code

File: index.html [hunk 3]
```diff
@@ -433,8 +505,19 @@
...
+        return `<div class="song-detail">${youtubeLink ? youtubeLink.id : ""} </div>`;
+      }*/
+
+      var songDetail = (youtubeLink) => (`<div class="song-detail">
+          <div class="songNav">
+            <div class="song_item song_number">${updateSongs}</div>
```
Comment (jcneally) [inline index.html:515]: This variable doesn't exist here

File: index.html [hunk 4]
```diff
@@ -448,10 +531,33 @@
           selectedSongIndex = $(this).index();
 
           //to select all song and then to remove the class from them
-                  $('.song-detail').removeClass('selected');
+          $('.song-detail').removeClass('selected');
           $(this).addClass('selected'); //to select clicked element
-      });
+        });
+
+        //var songsTotal = songList.length;
```
Comment (jcneally) [inline index.html:538]: remove unused code

File: index.html [hunk 5]
```diff
@@ -495,25 +609,24 @@
...
         	addSong(song);
         });
       }
 
       function createPlaylist() {
-
         window.location.href = './crossfade.html'
       }
 
+      function coutingSongs(){
```
Comment (jcneally) [inline index.html:626]: remove empty function that isn't used

File: index.html (hunk 2 above)
Comment (davigrca) [reply index.html:509]: I would rather to keep it because if I need it in the future for other projects

**Pull request merged**